src/api/models/api_response.py

```diff
@@ -0,0 +1,108 @@
+# coding: utf-8
+"""Assessment Model."""
+
+from __future__ import absolute_import
+
+# cisagov Libraries
+from api import util
+from api.models.base_model_ import Model
+
+# from datetime import date, datetime  # noqa: F401
+# from typing import Dict, List  # noqa: F401
+
+
+class ApiResponse(Model):
+    """NOTE: This class is auto generated by the swagger code generator program.
+
+    Do not edit the class manually.
```

Comment:
Capitalization and add a URL:
```suggestion
    """NOTE: This class is auto generated by the Swagger code generator program.
    https://swagger.io/tools/swagger-codegen/

    Do not edit the class manually.
```